File: app/src/main/java/com/example/trackerteacher/Item.java
package com.example.trackerteacher;

public class Item {
    String Name;
    String course;
    int image;
    String lastUpdateOfStatus;
    String status;
    String room;

    // constructor
    public Item(String Name, String course, int image, String lastUpdateOfStatus, String status, String room) {
        this.Name = Name;
        this.course = course;
        this.image = image;
        this.lastUpdateOfStatus = lastUpdateOfStatus;
        this.status = status;
        this.room = room;
    }

    // getter setter
    public String getName() {
        return Name;
    }

    public String getCourse() {
        return course;
    }

    public int getImage() {
        return image;
    }

    public String getLastUpdateOfStatus() {
        return lastUpdateOfStatus;
    }

    public String getStatus() {
        return status;
    }

    public String getRoom() {
        return room;
    }
}

File: app/src/main/java/com/example/trackerteacher/item.kt
package com.example.trackerteacher

/**
 * Represents a Professor's record retrieved from the RDBMS.
 * Used to display availability based on predefined schedules.
 */
data class Item(
    val name: String,               // Professor Name
    val course: String,             // Department/Program
    val image: Int,                 // Resource ID for profile picture
    val scheduleImage: Int = R.drawable.ic_launcher_foreground // Schedule image resource (default placeholder)
)